packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -31,6 +31,16 @@ function isTabItem(
   return 'value' in comp.props;
 }
 
+const NON_GROUP_TAB_KEY = '__noGroup__';
+function getValueFromSearchParams(groupId = NON_GROUP_TAB_KEY): string | null {
+  if (typeof window === 'undefined') {
+    return null; // Ignore during SSR.
+  }
+  const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Good point, replaced with react router instead.

Reply:
We have a dogfood section in our doc where you can add such test: that would be nice to add this test actually to check if it's well-supported.

I think it's not because you don't have any `useEffect(() => {},[qsValue])` that will resync the querystring value to state after mount

Reply:
> you can add such test
Not sure what exactly you mean, I have added a docs section to Tabs page which should be enough for dogfooding, but if not, just let me know.

> you don't have any useEffect(() => {},[qsValue]) that will resync the querystring value to state after mount

This is added now.